docs/apache-airflow/faq.rst

```diff
@@ -119,6 +119,18 @@ How do I trigger tasks based on another task's failure?
 
 You can achieve this with :ref:`concepts:trigger-rules`.
 
+When there are a lot of dags files, how to speed up parsing of new files?
```

Comment:
Should we be more specific to what is a lot?

Reply:
Updated